### ISSUE TITLE
Could the replacement paths specify a different target?

### ISSUE THREAD

**Issue by lopsided** [crossborne/gassetic#48]
Gassetic makes direct replacements in the replacement files. In my case none of the compiled assets are committed (or uncompiled vendors either), instead the CI server builds them again as part of the deploy. This means that my `index.html` contains `script` tags that are not referencing actual files in the repository, and are irrelevant anyway since they get replaced by the CI server's `gassetic build`. 

Ideally I would have something like this:

````
replacementPaths:
    - { from: base.html.twig, to: compiled.html.twig }
````

So the `base.html.twig` acts as a template and stays pretty static, and I could `git-ignore` `compiled.html.twig` to make a tidier repo!

Does this make sense? Any thoughts?

**Comment by kbsali** [comment]
Yep, that's a nice idea! It's somewhat annoying to have to undo all the changes made by assetic before commiting "real" changes.

**Comment by romanschejbal** [comment]
Yeah that sounds sensible. I am pretty busy these days but I will look at this later - if none else will?
